README.md

```diff
@@ -387,6 +387,25 @@ new L.GPX(url, {
 }).addTo(map);
 ```
 
+### Set which elements are shown on the map
+
+GPX file may contain tracks, routes and waypoints and in standard configurations
+all are shown.
+
+You can explicitly tell which elements to show by setting the `parseElements` flag to
```

Comment:
This is already described line 65. Maybe we can improve the section earlier in the README with the example you have below instead?